composer.json

```diff
@@ -52,6 +52,11 @@
     "extra": {
         "composer-exit-on-patch-failure": true,
         "enable-patching": true,
+        "patches": {
+            "drupal/core": {
+                "Deprecation warnings in Twig code cause test failures. https://www.drupal.org/project/drupal/issues/2600154": "https://www.drupal.org/files/issues/2018-08-13/2600154-3-60.patch"
```

Comment:
This patch should be included in openeuropa/drupal-core-require-dev and not directly on the theme. I've created the following issue for that: https://webgate.ec.europa.eu/CITnet/jira/browse/OPENEUROPA-1387